scp/templates/istio-auth/_helpers.tpl

```diff
@@ -1,17 +1,13 @@
 {{- define "shp.auth.issuer" -}}
-{{ printf "%s/realms/%s" ( include "shp.oidc.externalUrl" . ) ( .Values.keycloak.realm | default "tdf" ) }}
-{{- end -}}
-
-{{- define "shp.authinternal.issuer" -}}
-{{ printf "http://keycloak-http.%s.cluster.local/auth/realms/%s" ( .Values.keycloak.realm | default "tdf" ) }}
+{{- printf "%s/realms/%s" ( include "shp.oidc.externalUrl" . ) ( .Values.keycloak.realm | default "tdf" ) }}
```

Comment:
should the issuer also be internal when using the internal JWTURL? https://virtru.slack.com/archives/C03B9DR1631/p1685045820631169?thread_ts=1685034484.027449&cid=C03B9DR1631

Reply:
or is iss by default the external url

Reply:
should match whatever keycloak is using as its hostname - since that is what it used when it creates it's JWTs

Reply:
nvm looks like its kc uses the external url in the iss field